core/src/main/scala/org/apache/spark/sql/CachedDataFrame.scala

```diff
@@ -49,26 +52,38 @@ import org.apache.spark.unsafe.Platform
 import org.apache.spark.util.CallSite
 import org.apache.spark.{Logging, NarrowDependency, ShuffleDependency, SparkContext, SparkEnv, SparkException, TaskContext}
 
-class CachedDataFrame(df: Dataset[Row], val queryString: String,
+class CachedDataFrame(df: Dataset[Row], var queryString: String,
     cachedRDD: RDD[InternalRow], shuffleDependencies: Array[Int],
     val rddId: Int, val hasLocalCollectProcessing: Boolean,
     val allLiterals: Array[LiteralValue] = Array.empty,
     val allbcplans: mutable.Map[BroadcastHashJoinExec, ArrayBuffer[Any]] = mutable.Map.empty)
     extends Dataset[Row](df.sparkSession, df.queryExecution, df.exprEnc) with Logging {
 
   /**
-   * Return true if [[collectWithHandler]] supports partition-wise separate
-   * result handling by default, else result handler is invoked for a
-   * single partition result.
-   */
+    * Return true if [[collectWithHandler]] supports partition-wise separate
+    * result handling by default, else result handler is invoked for a
+    * single partition result.
+    */
```

Comment:
Check your fomatting settings -- I think original one was correct as per Spark and in codeStyleSettings.xml. Change IDEA Settings->Editor->Code Style to use "Project" scheme, close IDEA, copy codeStyleSettings.xml to .idea, then start.